0001-kaa-protocol/README.md

```diff
@@ -123,9 +123,9 @@ For example, use `/json` for JSON-formatted payload, and `/protobuf/<scheme_id>`
 
 ### Request/response pattern
 Many extensions require request/response style communication, which is not supported by MQTT natively.
-That is overcome by introducing a separate topic name for responses.
+That is overcome by introducing a separate resource path for responses.
```

Comment:
I'm also trying to come up with the language that will make it possible to refer to this section from other RFCs without getting into the MQTT vs CoAP vs other protocols specifics, which other protocols must be abstracted from.